src/Components/SimpleForm.js

```diff
@@ -0,0 +1,67 @@
+
+import React, { useState } from "react";
+import {
+  Form,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  ActionGroup,
+  Button,
+  CardBody,
+  Card,
+  Bullseye
+} from "@patternfly/react-core";
+import SimpleEmptyState from "./SimpleEmptyState";
+
+export default function SimpleForm(props) {
+  const [selectProductsVersion, setSelectProductsVersion] = useState("");
+  const [selectLocales, setSelectLocales] = useState("");
+  return (
+    <Bullseye>
+      <Card>
+        <CardBody>
+          <SimpleEmptyState />
+          <Form onSubmit={props.handleSubmit}>
```

Comment:
What might the reason to use a emptystate component inside a card?

Reply:
If someone uses the direct URL to land on a certain page then we show the message that you need to select options from the form to check the diff.

Using query params is a part of the next PR and we have already discussed that and conveyed it to @pravins